LiftLog.Ui/Store/Stats/StatsEffects.cs

```diff
@@ -88,7 +88,7 @@ state.Value.OverallViewSessionName is null
             var averageTimeBetweenSets = sessionsWithExercises
                 .SelectMany(x => x.RecordedExercises)
                 .SelectMany(x =>
-                    x.PotentialSets.Select(set => set.Set?.CompletionTime.ToTimeSpan())
+                    x.PotentialSets.Select(set => set.Set!.CompletionDate!.ToDateTime(set.Set!.CompletionTime!))
                         .WhereNotNull()
```

Comment:
```suggestion
                    x.PotentialSets.Select(set => set.Set)
                        .WhereNotNull()
                        .Select(set=>set.CompletionDate.ToDateTime(set.Set.CompletionTime!)
```

So these are still nullable, which may cause a crash if there are partially completed exercises.